options/options.html

```diff
@@ -196,6 +196,12 @@ <h4 class="i18n">options.otherSettings</h4>
             </ul>
             <div class="tab-content">
               <div id="urlsContent" class="tab-pane show active" role="tabpanel">
+                 <div class="input-group mb-3">
+                    <input  id="urlsSearchBar" type="search" Placeholder="Search" class="form-control searchBar"/>
+                    <span id="urlsSearchClear" class="form-control-feedback form-control-clear closeIcon">
+                        X
+                    </span>
```

Comment:
Better use an icon like in other places. Example:

```html
<i class="fas fa-backspace"></i>
```

Available icons that could be suitable: https://fontawesome.com/v5.9/icons?d=gallery&p=2&q=delete&m=free. I think that "backspace" or "eraser" could be the more appropriate.

With this change I think you will be able to simplify the css above too.